engine/verification/test/helper.go

```diff
@@ -398,108 +388,6 @@ func SetupMockConsensusNode(t *testing.T,
 	return &conNode, conEngine, wg
 }
 
-// SetupMockVerifierEng sets up a mock verifier engine that asserts the followings:
```

Comment:
Following test helpers 👇  are unused and won't be of any benefit in new architecture, hence removed.